src/css/app.sass

```diff
@@ -42,9 +42,6 @@ a
   line-height: 2rem
   user-select: none
 
-.shadow-2
```

Comment:
this override has been moved to scoped styles in the affected components. this class is from quasar and needs to be usable as intended